src/ng/directive/ngIf.js

```diff
@@ -39,7 +39,7 @@
  * @element ANY
  * @scope
  * @priority 600
- * @param {expression} ngIf If the {@link guide/expression expression} is falsy then
+ * @param {expression} ngIf If the {@link guide/expression expression} is false then
```

Comment:
it's not really a typo -- see https://developer.mozilla.org/en-US/docs/Glossary/Falsy